#Called Bingo Numbers 
#Send zero if you are cancelling the goal or send all the called numbers
int32[] calledBingoNumbers 
int32 newGameType
---
#Bingo Card State
# Card marked incorrectly = -1
# No bingo detected = 0
# Bingo detected = 1
int32 bingoCardState
int32[] missingNumbers
int32[] wrongNumbers
float32 cardLocationX
float32 cardLocationY
float32 cardLocationZ
---
#Detection state
# Bingo card not detected = 0
# Bingo card detected = 1
int32 cardDetected
